install-bls-signatures

```diff
@@ -13,16 +13,14 @@ if download_release_tarball tarball_path "${subm_dir}"; then
     tmp_dir=$(mktemp -d)
     tar -C "$tmp_dir" -xzf "$tarball_path"
 
-    cp -R "${tmp_dir}/include" .
-    cp -R "${tmp_dir}/lib" .
+    cp "${tmp_dir}/include/libbls_signatures.h" .
+    cp "${tmp_dir}/lib/libbls_signatures.a" .
+    cp "${tmp_dir}/lib/pkgconfig/libbls_signatures.pc" .
 else
     (>&2 echo "failed to find or obtain precompiled assets for ${subm_dir}, falling back to local build")
     build_from_source "${subm_dir}"
 
-    mkdir -p include
-    mkdir -p lib/pkgconfig
-
-    find "${subm_dir}" -type f -name libbls_signatures.h -exec mv -- "{}" ./lib/bls-signatures/include/ \;
-    find "${subm_dir}" -type f -name libbls_signatures_ffi.a -exec cp -- "{}" ./lib/bls-signatures/lib/ \;
-    find "${subm_dir}" -type f -name libbls_signatures.pc -exec cp -- "{}" ./lib/bls-signatures/lib/pkgconfig/ \;
+    find "${subm_dir}" -type f -name libbls_signatures.h -exec mv -- "{}" . \;
```

Comment:
Since you're changing this line, I suggest modifying `-exec mv -- "{}" . \;` to be `-exec cp -- "{}" . \;`.

Reply:
Will do.

Reply:
I didn't even notice it was `mv`.